ratelimits/limit.go

```diff
@@ -157,6 +169,31 @@ func parseOverrideNameId(key string) (Name, string, error) {
 	return name, id, nil
 }
 
+// parseOverrideNameEnumId is like parseOverrideNameId, but it expects the
+// key to be formatted as 'name:id', where 'name' is a Name enum string and 'id'
+// is a string identifier. It returns an error if either part is missing or invalid.
+func parseOverrideNameEnumId(key string) (Name, string, error) {
```

Comment:
This gets exercised in `DumpOverrides()`, but it'd be nice to see to see some test coverage for rejecting invalid overrides.